tests/Unit/inc/Engine/Preload/Controller/PreloadUrl/preloadUrl.php

```diff
@@ -81,6 +82,14 @@ public function testShouldDoAsExpected( $config ) {
 
 	protected function expectDesktopRequest( $config ) {
 		if ( $config['cache_exists'] ) {
+			$delay_value = 500000;
+
+			if ( isset( $config['rocket_preload_delay_between_requests'] ) ) {
+				$delay_value = $config['rocket_preload_delay_between_requests'];
+			}
+
+			Functions\expect('apply_filters')->with( 'rocket_preload_delay_between_requests', $delay_value )->andReturn( $delay_value );
```

Comment:
@jeawhanlee I am not sure I understand this line correctly. apply_filters is not expected to be called with `$delay_value` but with the default value 500000. In case it is change, this expectation would not match, would it?